plugins/accessibility/src/main.ts

```diff
@@ -18,15 +18,15 @@ const defaultAxeConfig: axe.Spec = {
   rules: [
     {
       id: 'color-contrast',
-      enabled: false
+      enabled: true
```

Comment:
Hey @pragsanshi! I think in this case we want to just make the default configuration an empty object (`{}`). That will turn on all Axe rules by default, which is probably what users of this plugin will expect.

Reply:
Thanks for contributing!